api/Request.json

```diff
@@ -571,74 +449,30 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Request/clone",
           "spec_url": "https://fetch.spec.whatwg.org/#ref-for-dom-request-clone①",
           "support": {
-            "chrome": [
-              {
-                "version_added": "42"
-              },
-              {
-                "version_added": "41",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "Experimental Web Platform Features"
-                  }
-                ]
-              }
-            ],
+            "chrome": {
+              "version_added": "42"
+            },
             "chrome_android": {
               "version_added": false
```

Comment:
I don't believe that this isn't supported on Chrome for Android, but this PR doesn't do anything with the entry, so OK to leave it for later.

Reply:
Submitted a PR to correct this!  #10565